projects/elements/src/lib/lazy-elements/lazy-elements.module.ts

```diff
@@ -55,16 +79,29 @@ export class LazyElementsModule {
     lazyElementsLoaderService: LazyElementsLoaderService,
     @Optional()
     @Inject(LAZY_ELEMENT_CONFIGS)
-    elementConfigsMultiProvider: ElementConfig[][]
+    elementConfigsMultiProvider: ElementConfig[][],
+    @Optional()
+    @Inject(LAZY_ELEMENT_GLOBAL_OPTIONS)
+    globalOptions: LazyElementModuleGlobalOptions
   ) {
     if (elementConfigsMultiProvider && elementConfigsMultiProvider.length) {
       const lastAddedConfigs =
         elementConfigsMultiProvider[elementConfigsMultiProvider.length - 1];
       lazyElementsLoaderService.addConfigs(lastAddedConfigs);
     }
+
+    if (globalOptions) {
+      lazyElementsLoaderService.setGlobalOptions(globalOptions);
+    }
   }
 }
 
 export interface LazyElementModuleOptions {
   elementConfigs?: ElementConfig[];
 }
+
+export interface LazyElementModuleGlobalOptions {
```

Comment:
I think I see what was the intention in here but the original `LazyElementModuleOptions ` was introduced exactly to prevent the need to introduce additional parameters of `forRoot` / `forFeature`

so then we want to stay with 
```
static forRoot(options: LazyElementModuleOptions)
```

and the `LazyElementModuleOptions` can get additional optional `global`

```
export interface LazyElementModuleOptions {
  elementConfigs?: ElementConfig[];
  globalOptions: LazyElementModuleGlobalOptions 
}
```

Reply:
As to address how to prevent people passing this to a `forFeature`, as they shouldn't, we can hope that this will be pretty much clear `root` => `global` or you see, we can call it `rootOptions` instead and throw error if they pass it in the `forFeature`...

That way the naming is obvious and the check is in place to enforce it, what are your thoughts ?

Reply:
Addresses 👍